index.html

```diff
@@ -395,6 +395,25 @@ <h2>Convert a <var>name</var> to a <var>timestamp</var></h2>
         The <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface was defined in [[NAVIGATION-TIMING]] and is now considered obsolete. The use of names from the <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface is supported to remain backwards compatible, but there are no plans to extend this functionality to names in the <a data-cite="NAVIGATION-TIMING-2#dom-performancenavigationtiming">PerformanceNavigationTiming</a> interface defined in [[NAVIGATION-TIMING-2]] (or other interfaces) in the future.
       </p>
     </section>
+  </section><section id="recommended-mark-names" class="informative">
+    <h2>Recommended Mark Names</h2>
+    <p>Developers are encouraged to use the following Recommended Mark Names to
+      mark common timings. The user agent is responsible for storing a new
+      timestamp under the specified mark name for Recommended Mark Names, just
```

Comment:
I don't know that we need to capitalize that "Recommended Mark Names". If we want this to be a concept, it might be worthwhile to `<dfn>` it once, and then link to it.

Reply:
Remove capitalization.